### DISCUSSION TITLE
Error: jq: error (at <stdin>:1): Cannot iterate over null (null)

### DISCUSSION THREAD

**Discussion by Yosh82** [RandomNinjaAtk/docker-lidarr-extended#245] [General]
Hi,

I keep seeing this error in the LE logs. It still seems to be downloading ok, so unsure if it's causing issue or not, but nothing obvious:

![image](https://github.com/RandomNinjaAtk/docker-lidarr-extended/assets/106485860/15202b52-cc35-4160-8260-00066e97c707)

Is anyone else seeing these?

Thanks.

**Comment by RandomNinjaAtk** [comment]
You're probably being throttled by musicbrainz. Not much can be done to fix that...

**Reply by Yosh82** [reply]
I did see the other post about that, wasn't sure if it was related or not. Thanks for the response.

**Reply by Ozai77** [reply]
This looks like it's due to the agent. I changed the agent in the scripts and it works for me now. Not sure why it's getting rejected.

**Reply by RandomNinjaAtk** [reply]
Maybe too many with same agent. What did you change it to? 
Maybe I can randomize it to resolve the issue.

**Reply by Ozai77** [reply]
I changed it to just 'lidarr'. I'm guessing that it would work with something else, but stopped trying to solve the issue after the api calls started working.

**Reply by RandomNinjaAtk** [reply]
The latest changes should resolve it

**Reply by Yosh82** [reply]
Thanks guys. It's working fine now 👍

**Comment by RandomNinjaAtk** [comment]
Based on comments, these changes may fix the issue:
https://github.com/RandomNinjaAtk/docker-lidarr-extended/commit/be6822a773887a0cada7cdcff641340bd116b08f
https://github.com/RandomNinjaAtk/docker-lidarr-extended/commit/d1c961d2a68051446e0b3f7905620b3f27438be3
https://github.com/RandomNinjaAtk/docker-lidarr-extended/commit/5401a46aee86307e48c0119ef1392d292cc047ad

But as of now, it is untested.... After you have pulled the updates, see if that resolves the issue...

**Reply by RandomNinjaAtk** [reply]
I had to make one more round of changes, but I think its resolved.... At least based on my testing...
